app/models/manageiq/providers/redhat/infra_manager.rb

```diff
@@ -119,4 +119,23 @@ def remove_disks(disks, vm)
       disks.each { |disk_id| service.attachment_service(disk_id).remove }
     end
   end
+
+  def vm_migrate(vm, options = {})
```

Comment:
I was expecting `migrate` to be a VM operation.  Any reason that this is on the infra manager instead?